### DISCUSSION TITLE
How to disable the "Create Room" button on the homepage

### DISCUSSION THREAD

**Discussion by normanwankowski** [mozilla/hubs#4869] [Hubs Cloud Questions] [answered]
How can I disable to CREATE ROOM button on the homepage ?

It creates a lot of expensive data storage and is confusing for the members of my community.
We prefer to just have selected members be able to create rooms. All others are meeting in Public rooms, where they can also pin objects.

Is there a simple way to just hide the Create Room button on the homepage?

**Comment by rawnsley** [comment]
You probably want _Disable room creation_ in **Setup** > **App Setting** in the administration pages:

<img width="1004" alt="Screenshot 2021-11-29 at 09 36 31" src="https://user-images.githubusercontent.com/303516/143843475-a220b68f-9fad-450b-9c45-fbbd3fcf1a8a.png">

**Comment by normanwankowski** [comment]
But in the case the selected creators can't create rooms. And I also can't make them Administrators of my system. I was hoping I can simply hide the button.

**Comment by rawnsley** [comment, marked as answer]
If you have your own Hubs Cloud you can hide the button. It's [here](https://github.com/mozilla/hubs/blob/5369ce260da08d726eddb05983430a753355335c/src/react-components/home/HomePage.js#L86) in the code.

You might be able to hide it using CSS if you inject it into the page header via **Setup** > **Server Settings** > **Extra Homepage Header HTML**. This would mean no source code changes.
